source/binary.cpp

```diff
@@ -25,9 +25,9 @@
 #include "assembly_grammar.h"
 #include "diagnostic.h"
 #include "ext_inst.h"
+#include "latest_version_spirv_header.h"
```

Comment:
This header should be included before other local headers. Please also add blank line, so that clang-format doesn't reorder.

Reply:
Same